test/boost/chunked_vector_test.cc

```diff
@@ -17,6 +17,7 @@
 #include <random>
 #include "utils/chunked_vector.hh"
 #include "utils/amortized_reserve.hh"
+#include "types/types.hh"
```

Comment:
Why is a container test using the database type system?

Reply:
I wanted the test case to be interesting and realistic so I used `initializer_list<data_value>`.
It's not must to have if we don't want that dependency.

Reply:
I usually try to avoid such dependencies in tests because 1. it obscures what is actually being tested (when I read your test, I had to recall what "data_value" is, but it isn't important for this test), and 2. if somebody ever modifies data_value, it can break the irrelevant test for chunked_vector.
But in this case it seems minor to me, and I don't insist on changing this. So it's Avi's call if he want's to insist here.

Reply:
It's anti-modular. A module that provides a container shouldn't depend on a module that is several levels above it.